database/init_db.production/regions.sql

```diff
@@ -0,0 +1,280 @@
+DROP TABLE IF EXISTS regions;
+
+CREATE TABLE IF NOT EXISTS regions (
+    ogc_fid int GENERATED BY DEFAULT AS IDENTITY,
+    name text,
+    country text,
+    geom geometry,
+    subjective_quality_expectations text,
+    degree_of_urbanisation_class int,
+    degree_of_urbanisation_name text,
+    shdi float
+);
+
+
+/* Select custom OQT regions from NUTS and GADM datasets */
```

Comment:
I think for now it is okay to do it like this, ideally this would be a bit more structured e.g. in a table so that we can get an easier overview on the attributes such as subjective_quality_expectations.

I'm just thinking about how we would extend this. But let's think about it, when we have many more regions. When adding only a few more once in a while, this file will work.

Reply:
A better overview would be nice. We could extent the CLI command `list-regions` to output a csv with all the attributes. I will open an issue for that.